### ISSUE TITLE
[Feature Request?] Missing labels on some icons

### ISSUE THREAD

**Issue by jmfergeau** [vrcx-team/VRCX#1785]
One of my friends is blind, and she started to play VRC with me and my friends. VRCX is simply a lifesaver, as it makes some operations easier for her with her screen reader. (Such as joining or making invites, sending and receiving boops, etc)

Though some buttons are still missing labels for screen readers and appear as "empty" for them. Especially the buttons that are only shown as icons, such as the favorite buttons, the notification button, etc.

It would be great to add labels on these buttons. Maybe by using aria labels, as it seems to be an electron app? (unless I'm mistaking?)

**Comment by Natsumi-sama** [comment]
It's Cef/Electron Vue web app, but a PR adding these in would be very much welcome, if you need help finding the right files I'd be happy to help on Discord.
